src/api/messagesFlags.js

```diff
@@ -1,14 +1,16 @@
-import { apiPost, Auth } from './apiFetch';
+/* @flow */
+import { Auth } from '../types';
+import { apiPost } from './apiFetch';
 
 export default (
   auth: Auth,
   messages: number[],
   op: string,
   flag: string,
-): number[] =>
+): mixed =>
```

Comment:
👍 to this

We can also leave it like this for now and add a TODO to come back to it if it's tricky